src/jdk.compiler/share/classes/com/sun/source/util/DocTreeFactory.java

```diff
@@ -175,6 +176,15 @@ DocCommentTree newDocCommentTree(List<? extends DocTree> fullBody,
      */
     ErroneousTree newErroneousTree(String text, Diagnostic<JavaFileObject> diag);
 
+    /**
+     * Creates a new {@code EscapeTree} object, to represent an escaped character.
+     * @param ch the character
+     * @return an {@code EscapeTree} object
+     *
+     * @since 21
+     */
+    EscapeTree newEscapeTree(char ch);
```

Comment:
On `DocTreeFactory`
1. at the time the API was created, it was modeled on the javac AST, and yes, while it might be a bit weird here, it's less weird than the one for javac.
2. `DocTreeFactory` is used in the doclet to create comments for mandated methods, so one way or another, it needs to be public.